packages/react/package.json

```diff
@@ -5,12 +5,11 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "yarn clean && tsc && yarn copy-assets && yarn copyfiles -u 1 src/**/*.css dist/",
-    "clean": "rimraf dist",
-    "copy-assets": "cp -r src/lib/assets dist/lib",
-    "format": "eslint --fix . && prettier 'src/**/*.ts' --write --ignore-path .gitignore",
-    "lint": "eslint --ignore-path .eslintignore .",
-    "test": "jest --passWithNoTests"
+    "prebuild": "rimraf dist",
+    "build": "yarn build:types && yarn build:js && yarn build:prettier",
+    "build:types": "tsc --project tsconfig.declarations.json",
+    "build:js": "babel src --out-dir dist --extensions '.js,.jsx,.ts,.tsx' --source-maps --copy-files",
+    "build:prettier": "prettier --write dist"
```

Comment:
do we need to do this on the compiled build? seems a bit unnecessary to make the files in dist prettier?

Reply:
This takes like a few seconds max, and if not the babel puts them all into one line so then if we inspect via a debug console it won't be formatted. Not necessary, just nice to have and inexpensive.

Reply:
that's fine, just wanted to make sure that it doesn't add to the build size